CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -14,6 +14,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ### etcd server
 - Add [`etcd --max-concurrent-streams`](https://github.com/etcd-io/etcd/pull/14219) flag to configure the max concurrent streams each client can open at a time, and defaults to math.MaxUint32.
+- Add [`etcd --experimental-compact-hash-check-enabled=true --experimental-compact-hash-check-time=1m``](https://github.com/etcd-io/etcd/issues/14039) flags to enable reliable corruption detection by periodicly checking hashes on compacted revisions.
```

Comment:
This is the change log item instead of a guide/suggestion, I think we should reflect the real design/implementation. 

Note that the default value for `--experimental-compact-hash-check-enabled` is `false`.

Reply:
Done.